src/main/java/teammates/ui/controller/InstructorFeedbackQuestionEditAction.java

```diff
@@ -68,12 +62,20 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                                             .getInstructorFeedbackEditLink(courseId, feedbackSessionName));
     }
 
-    private void deleteQuestion(FeedbackQuestionAttributes updatedQuestion) {
-        logic.deleteFeedbackQuestion(updatedQuestion.getId());
+    private void deleteQuestion() {
+        String questionId = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_QUESTION_ID);
+
+        String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
+
+        String feedbackSessionName = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
+
```

Comment:
Can remove the 2 blank lines in between the last 3 parameters. Rationale: these 3 parameters are used together and form one logical block. Also, it'd be nice if the order was the same as where the parameters are used (i.e. `questionNumber` then `feedbackSessionName` then `courseId`).

Reply:
👍 done!